articles/quickstart/spa/angular/02-calling-an-api.md

```diff
@@ -116,6 +116,10 @@ import { HttpClient } from '@angular/common/http';
 // Import AuthService from the Auth0 Angular SDK to get access to the user
 import { AuthService } from '@auth0/auth0-angular';
 
+// Import Pluck from RXJS 
+import { pluck } from 'rxjs/operators';
```

Comment:
Can we add this to the existing list on line 111 so we get `import { concatMap, tap, pluck } from 'rxjs/operators';`